api/export.go

```diff
@@ -278,11 +278,11 @@ func copyDirToExportWriter(writer ExportWriter, inPath string, outPath string) *
 }
 
 func ExportLocalStorage(writer ExportWriter, options *ExportOptions, teamId string) *model.AppError {
-	teamDir := utils.Cfg.ServiceSettings.StorageDirectory + "teams/" + teamId
+	teamDir := utils.Cfg.ImageSettings.Directory + "teams/" + teamId
```

Comment:
If we're doing a bunch of renaming of config file stuff, I think it might be good to change `ImageSettings` to `FileSettings`. 0/5 though